engine/class_modules/apl/warlock.cpp

```diff
@@ -389,8 +389,8 @@ void destruction( player_t* p )
   precombat->add_action( "variable,name=disable_cb_2t,value=variable.do_rof_2t|variable.allow_rof_2t_spender>0.01&variable.allow_rof_2t_spender<0.99" );
   precombat->add_action( "grimoire_of_sacrifice,if=talent.grimoire_of_sacrifice.enabled" );
   precombat->add_action( "snapshot_stats" );
+  precombat->add_action( "cataclysm,if=active_enemies>=2&raid_event.adds.in>15" );
```

Comment:
Precast cata as it is first a DPS gain, and second, it helps with cdf gain, solving an issue when we were casting cdf before cata. And in general, we are always opening with cata, so it actually reflects reality.